packages/bigtable/test/index.js

```diff
@@ -18,43 +18,39 @@
 
 var assert = require('assert');
 var extend = require('extend');
-var format = require('string-format-obj');
 var googleProtoFiles = require('google-proto-files');
 var nodeutil = require('util');
 var proxyquire = require('proxyquire');
 var sinon = require('sinon').sandbox.create();
 
-var GrpcService = require('@google-cloud/common').GrpcService;
+var common = require('@google-cloud/common');
+var Cluster = require('../src/cluster.js');
+var Instance = require('../src/instance.js');
 var PKG = require('../package.json');
-var Table = require('../src/table.js');
-var util = require('@google-cloud/common').util;
 
-var fakeUtil = extend({}, util);
-
-function FakeGrpcService() {
-  this.calledWith_ = arguments;
-  GrpcService.apply(this, arguments);
-}
-
-nodeutil.inherits(FakeGrpcService, GrpcService);
-
-function FakeTable() {
-  this.calledWith_ = arguments;
-  Table.apply(this, arguments);
+var fakeUtil = extend({}, common.util);
+var fakeStreamRouter = {
+  extend: function() {
+    this.calledWith_ = arguments;
+  }
+};
+
+function createFake(Class) {
+  function Fake() {
+    this.calledWith_ = arguments;
+    Class.apply(this, arguments);
+  }
+  nodeutil.inherits(Fake, Class);
+  return Fake;
 }
 
-function FakeFamily() {}
+var FakeGrpcService = createFake(common.GrpcService);
+var FakeCluster = createFake(Cluster);
+var FakeInstance = createFake(Instance);
+var FakeGrpcOperation = createFake(function() {});
```

Comment:
@stephenplusplus this is obviously the wrong way to test this (I think) - how do we test companion `common` features in the future?